tests/collecting/CollectorsTest.py

```diff
@@ -264,6 +264,13 @@ def test_no_duplications(self):
             ['kind'],
             self.log_printer)[0]), 2)
 
+    def test_bear_suffix(self):
+        self.assertEqual(len(collect_bears(
```

Comment:
this is shocking indentation.
(yes I know that it is used above, but no need to repeat it)

Reply:
since there are 3 opening round brackets you want me to have 3 indents after this line? @jayvdb 
`self.assertEqual(len(collect_bears(
                    os.path.join(
                        self.collectors_test_dir,
                        'bears'),
                    ['namebear'], ['kind'],
                    self.log_printer)[0]), 1)`